activesupport/test/core_ext/time_with_zone_test.rb

```diff
@@ -482,22 +482,23 @@ def test_freeze_preloads_instance_variables
   end
 
   def test_method_missing_with_non_time_return_value
-    @twz.time.expects(:foo).returns('bar')
+    @twz.time.class.send(:define_method, :foo, -> { 'bar' })
```

Comment:
:smile:  Not familiar with this way. This is reporting syntax error though. Can you please provide some reference about it (how it works) so I can learn from there?